package.json

```diff
@@ -80,7 +82,10 @@
         "@types/request": "^2.48.7",
         "@types/retry": "^0.12.1",
         "@types/semver": "^7.3.8",
+        "@types/socket.io-client": "^1.4.36",
+        "@types/socket.io-parser": "^2.2.1",
```

Comment:
I think it will be much better to just remove socket.io module and only use websocket. All custom pricer users will need to update their script to use websocket.

(remove `"@tf2autobot/socket.io-client": "^1.0.4"` too)

Reply:
Okay. I can do that. Thanks for the review!

Reply:
This was addressed